src/Kafka/KafkaInstanceStatus/components/StatusLabel.stories.tsx

```diff
@@ -156,3 +162,27 @@ suspended.parameters = {
     },
   },
 };
+
+export const suspending = Template.bind({});
```

Comment:
Not a huge deal but could you please do as the action suggests?

Reply:
done